src/main/kotlin/com/rooplor/classcraftbackend/configuration/OpenApiConfig.kt

```diff
@@ -0,0 +1,37 @@
+package com.rooplor.classcraftbackend.configuration
+
+import io.swagger.v3.oas.annotations.OpenAPIDefinition
+import io.swagger.v3.oas.annotations.enums.SecuritySchemeIn
+import io.swagger.v3.oas.annotations.enums.SecuritySchemeType
+import io.swagger.v3.oas.annotations.info.Contact
+import io.swagger.v3.oas.annotations.info.Info
+import io.swagger.v3.oas.annotations.security.SecurityScheme
+import io.swagger.v3.oas.annotations.servers.Server
+
+@OpenAPIDefinition(
+    info = Info(
+        contact = Contact(
+            name = "Rooplor",
+            email = "rooplorgit@gmail.com"
+        ),
+        description = "The ClassCraft API definition",
+        title = "The ClassCraft API",
+        version = "1.0"
+    ),
+    servers = arrayOf(
+        Server(
+            description = "Local ENV",
+            url = "http://localhost:8080"
+        )
+    )
+)
+//@SecurityScheme(
+//    name = "bearerAuth",
+//    description = "JWT auth description",
+//    scheme = "bearer",
+//    type = SecuritySchemeType.HTTP,
+//    bearerFormat = "JWT",
+//    `in` = SecuritySchemeIn.HEADER
+//)
```

Comment:
What is this. Why it was commented

Reply:
Can we enable it already. I think we shouldn't merge pr that has disabled code. It's use them or delete them

Reply:
sure on prob let enable it